README.md

```diff
@@ -1,7 +1,41 @@
 # help-center-content
+
 Repository assembling VTEX's Help Center content
 
+## In this repository
+
+You will find the following folders in this repository:
+
+-TBD
+
+## Contributing with Help Center Portal documentation
+
+We're so glad you're here! Thanks for being interested.
+
+### How do I add a download file to my article?
```

Comment:
The step by step is good. But I think it could be helpful to include the file URL template. I guess it is `https://github.com/vtexdocs/help-center-content/raw/main/{file-path}` (URL encoded), but it's worth checking.

Reply:
In this case, it might be better to just leave the "address link" reference as it is currently described, as this is a raw link (specific to the automatic download functionality). It will also sometimes include characters like "%20" and not the direct path.